<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditPropertyDialog</class>
 <widget class="QDialog" name="AddEditPropertyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="propertyNameWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="propertyNameLabel">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="propertyNameField"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mucalculusTab">
      <attribute name="title">
       <string>Mu-calculus</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="mcfLabel">
         <property name="text">
          <string>Mu-calculus formula:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="mcrl2::gui::qt::CodeEditor" name="formulaTextField">
         <property name="placeholderText">
          <string>Type your mu-calculus formula here</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="equivalenceTab">
      <attribute name="title">
       <string>Equivalence</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="equLabel1">
         <property name="text">
          <string>The process:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="mcrl2::gui::qt::CodeEditor" name="initTextField1">
         <property name="placeholderText">
          <string>Type your process expression here</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="equLabel2">
         <property name="text">
          <string>is equivalent under:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="EquivalenceComboBox" name="equivalenceComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="equLabel3">
         <property name="text">
          <string>to the process:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="mcrl2::gui::qt::CodeEditor" name="initTextField2">
         <property name="placeholderText">
          <string>Type your process expression here</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="buttonsWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="parseLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveAndParseButton">
        <property name="text">
         <string> Save and Parse </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveAndCloseButton">
        <property name="text">
         <string> Save and Close </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EquivalenceComboBox</class>
   <extends>QComboBox</extends>
   <header>addeditpropertydialog.h</header>
  </customwidget>
  <customwidget>
   <class>mcrl2::gui::qt::CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>mcrl2/gui/codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
